doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -189,6 +189,19 @@ KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
 Bluetooth LE
 ============
 
+.. rst-class:: v2-9-0-nRF54H20-rc1 v2-9-0 v2-8-0
+
+NCSDK-31528: Deadlock on sysworkq with ``tx_notify`` in Host
+  If a blocking callback (which in turn needs to trigger an HCI command to complete its task) is run on the system workqueue when the HCI command buffer pool is full, the TX processing code will get blocked.
+  This prevents TX command buffers from being freed, deadlocking the application.
```

Comment:
```suggestion
  This situation prevents TX command buffers from being freed, deadlocking the application.
```

Otherwise it is not clear what does "this" refers to.

Reply:
IMO it's obviously referring to the statement/line above, and adding "situation" doesn't really contribute (to the situation :)).